cv32e40s/sim/ExternalRepos.mk

```diff
@@ -15,7 +15,7 @@ export SHELL = /bin/bash
 
```

Comment:
Does it pass ci_check, and is formal still runnable?

Reply:
`ci_check` is suffering from a problem atm, but here is a reference of a run I did recently (pic below), so I would expect that the same tests pass/fail for this PR:

![image](https://user-images.githubusercontent.com/75431602/220904166-b827f201-feb3-49a0-996b-f4b493783fd8.png)

`Error: Zcm* is not compatible with the `c' extension`

`Error: illegal operands `cm.push {ra},-64'`

Reply:
I'll give ci_check and formal a go, and report when they're done.

Reply:
@silabs-robin Which tool is producing this 'Error: Zcm* is not compatible with the c' extension`' and why?

Reply:
> Which tool is producing this 'Error: Zcm* is not compatible with the c' extension`' and why?

`gcc` / `gas`, because we have used `.option norvc` and particularly `.option rvc`, and it is now in conflict with the `march` containing `*_zcb_zcmp`.

This is easy to fix (using `.option push` and `.option pop` around a section requiring `.option norvc`) for our directed tests, for our own random tests, and our bsp, but I think riscv-dv also uses this and that is more tricky. Henrik indicated he might have an idea for how to fix that last one.

Reply:
ci_check gives the same number of passed/failed, but with a few variations to which tests fail. The error seems consistent with the Zcm* mentioned above though.

Reply:
Formal seems ok